src/site/_includes/components/Img.js

```diff
@@ -93,6 +92,21 @@ const Img = function (args) {
     imgTag = html`<a href="${fullSrc}">${imgTag}</a>`;
   }
 
+  if (caption) {
+    imgTag = html`${imgTag}
+      <figcaption class="w-figcaption">
+        ${caption === true ? alt : caption}
+      </figcaption>`;
+  }
+
+  if (figure || caption) {
+    imgTag = html`<figure
+      class="w-figure ${figure === true ? '' : `w-figure--${figure}`}"
```

Comment:
```js
`${figure === true ? '' : `w-figure--${figure}`}`
```

I think this is trying to let folks pass a modifier class through but I don't think it works because it's doing a strict equality check for `true`. Also if someone provides a caption and no `figure` it results in `w-figure--undefined`.

![image](https://user-images.githubusercontent.com/1066253/108290302-f1994280-7144-11eb-846d-7c1eb3ca1375.png)